Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -1293,20 +1275,15 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <SubType>Designer</SubType>
     </Content>
-    <ZipProject Include="ProjectTemplates\AzureNodejsApp\ChangeConfig.ps1" />
-    <ZipProject Include="ProjectTemplates\AzureNodejsApp\download.ps1" />
     <ZipProject Include="ProjectTemplates\AzureNodejsApp\AzureNodejsApp.njsproj">
       <SubType>Designer</SubType>
     </ZipProject>
     <ZipProject Include="ProjectTemplates\AzureNodejsApp\package.json" />
     <ZipProject Include="ProjectTemplates\AzureNodejsApp\README.md" />
-    <ZipProject Include="ProjectTemplates\AzureNodejsApp\setup_web.cmd" />
     <ZipProject Include="ProjectTemplates\AzureNodejsApp\Web.Debug.config">
       <SubType>Designer</SubType>
     </ZipProject>
     <ZipProject Include="ProjectTemplates\AzureNodejsApp\Web.config" />
-    <ZipProject Include="ProjectTemplates\AzureNodejsApp\node.cmd" />
-    <ZipProject Include="ProjectTemplates\AzureNodejsApp\iisnode.yml" />
```

Comment:
Why are you leaving the iisnode.yml file in some templates, and not others? Is it needed in some?